AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/physx_collider.py

```diff
@@ -0,0 +1,318 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import azlmbr.physics as physics
+import azlmbr.math as math
+
+from editor_python_test_tools.editor_entity_utils import EditorEntity
+from editor_python_test_tools.asset_utils import Asset
+
+COMPONENT_NAME = "PhysX Collider"
```

Comment:
suggestion: Should this be moved to the physics const file? (also consider naming the file `physics_constants` as this will make it much easier to search for. Also is there a way in Python to ensure those strings are immutable or is that the default? Thanks!

Reply:
That's how I originally had things. But as I've moved forward with the component behavior classes I've considered moving them into their behavior classes and the pattern to be to find these values in these classes.  

However, there could be value keeping them central in case somebody ever wants these values but not use these behavior classes.

I think I'll wind up actually keeping the Physics consts file holding these.

Reply:
Okay sounds good thanks :+1: